dace/libraries/sparse/nodes/__init__.py

```diff
@@ -1,2 +1,3 @@
 # Copyright 2019-2022 ETH Zurich and the DaCe authors. All rights reserved.
 from .csrmm import CSRMM
+from .csrmv import CSRMV
```

Comment:
Missing newline at the end. To clarify, it is a styling rule to have a new line at the end of each file.